docs/design/sno-board.md

```diff
@@ -0,0 +1,50 @@
+# SNOBoard
+
+## OVERVIEW
+
+The SNOBoard is a GUI for Storage Node Operators to get more information about their storage nodes performance as it pertains to the network or individual satellites.
+
+Wireframes:
+https://www.figma.com/file/IlY5UNF94rEpCLGR6BeaOhXX/Local-Storage-Node-Dashboard-Low-Fidelity-Mock?node-id=7%3A520 
+
+## GOALS
+
+- Give SNOs a pretty graphical user interface they can look at to get metrics on:
+	- Node(s) performance per satellite
+	- Storage space
+  	- Bandwidth
+	- Audit information
+	- Uptime information
+  	- Current Status
+	- SN version currently running
+	- DQ’ed status
+	- Containment status
+- Satellite notifications
+	- Notifications center on the SNOBoard
+	- Email notification opt-in
+- Expected payout per satellite, per node
+- Give SNOs a GUI to configure their SNs
+	- Ability to adjust allocated egress/ storage space in the GUI
+	- Ability to graceful exit a satellite or the network in the GUI
+
+## Business Requirements/ Job Stories
+
+- When a SNO starts the CLI dashboard I want the URL for the SNOBoard to be printed on the screen so that the user can easily access the SNOBoard.
+- When my SN is connected to multiple satellites I want the ability to select a satellite on the SNOBoard so that I can drill down into information about my node as it pertains to that satellite.
+	- All satellites should be the default option
+		- This option should NOT show information about uptime or audit checks because those are specific to satellites
+		- The “All Satellites” option should only show information about allocated and used storage space/ bandwidth 
```

Comment:
Can we also estimate the total payout that will come from all satellites?

Also, we can display the total amount held in the escrow of all satellites, if we decide to provide this info (see the previous comment).